real_check/CMakeLists.txt

```diff
@@ -1,4 +1,6 @@
 #Copyright (c) Microsoft. All rights reserved.
 #Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
-add_subdirectory(test)
+if(${run_unittests})
+    add_subdirectory(test)
+endif()
```

Comment:
Let's add a new line at the end of file.

Reply:
Done.